src/geo_mag_declination.cpp

```diff
@@ -34,9 +34,8 @@
 /**
 * @file geo_mag_declination.c
```

Comment:
While you're at it, could you fix the comment for the filename?

Reply:
Done.